common/src/contract-details.ts

```diff
@@ -9,46 +8,7 @@ import {
 import { richTextToString } from './util/parse'
 import { getCpmmProbability } from './calculate-cpmm'
 import { getDpmProbability } from './calculate-dpm'
-import { formatMoney, formatPercent } from './util/format'
-import { filterDefined } from './util/array'
-import { DOMAIN } from './envs/constants'
-
-export function contractMetrics(contract: Contract) {
-  // eslint-disable-next-line @typescript-eslint/no-var-requires
-  const dayjs = require('dayjs')
-  const { createdTime, resolutionTime, isResolved } = contract
-
-  const createdDate = dayjs(createdTime).format('MMM D')
-
-  const resolvedDate = isResolved
-    ? dayjs(resolutionTime).format('MMM D')
-    : undefined
-
-  const volumeLabel = `${formatMoney(contract.volume)} bet`
-
-  return { volumeLabel, createdDate, resolvedDate }
-}
-
-// String version of the above, to send to the OpenGraph image generator
-export function contractTextDetails(contract: Contract) {
-  // eslint-disable-next-line @typescript-eslint/no-var-requires
-  const dayjs = require('dayjs')
-  const { closeTime, groupLinks } = contract
-  const { createdDate, resolvedDate, volumeLabel } = contractMetrics(contract)
-
-  const groupHashtags = groupLinks?.map((g) => `#${g.name.replace(/ /g, '')}`)
-
-  return (
-    `${resolvedDate ? `${createdDate} - ${resolvedDate}` : createdDate}` +
-    (closeTime
-      ? ` • ${closeTime > Date.now() ? 'Closes' : 'Closed'} ${dayjs(
-          closeTime
-        ).format('MMM D, h:mma')}`
-      : '') +
-    ` • ${volumeLabel}` +
-    (groupHashtags ? ` • ${groupHashtags.join(' ')}` : '')
-  )
-}
+import { formatPercent, formatWithCommas } from './util/format'
```

Comment:
If we are sticking with Vercel and using it for stuff like this, we're probably going to have to figure out some general strategy for keeping track of "which of our code can be run in edge functions".